### PR TITLE
run tests for typescript 3.7

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

run types tests for
- current typescript version
- 3.8.3 version

Updated TypeScript version from 3.7.5 to 3.8.3

## Types of changes

- [x] internal

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

### CODE REVIEW

File: tests/typings/copy.js
Comment (christian-bromann) [inline tests/typings/copy.js:51]:
```suggestion
const TS_PARAMETER = '--ts='
let tsVersion = process.argv.find((v) => v.startsWith(TS_PARAMETER))
if (tsVersion) {
    tsVersion = tsVersion.substr(TS_PARAMETER.length)
}
```

File: tests/typings/@typescript/ts3.7/package.json
Comment (christian-bromann) [inline tests/typings/@typescript/ts3.7/package.json:8]:
```suggestion
```

**Review by christian-bromann — APPROVED** [review]
👍

**Review by christian-bromann — COMMENTED** [review]
Maybe we rename the copy script to `setup` given that it now also installs dependencies. I will leave it up to you, very minor.

**Comment by mgrybyk** [comment]
Please don't merge, tests should be fixed first

**Comment by mgrybyk** [comment]
@christian-bromann sorry to say I had to upgrade the minimum TypeScript version to 3.8.3 because there are too many issues with 3.7.5. To be clear, existing types won't work in 3.7.5 

Also, had to remove `@ts-expect-error` from tests as far as it's a feature of `3.9.x`

**Comment by mgrybyk** [comment]
@christian-bromann some wdio cli test ia failing

**Comment by christian-bromann** [comment]
weird, only on Windows 😠

**Comment by christian-bromann** [comment]
but it seems to be also broken in master, so I will merge this and see if I can fix it.

**Pull request merged**
